common/frame.go

```diff
@@ -1590,7 +1588,10 @@ func (f *Frame) Title() string {
 	f.log.Debugf("Frame:Title", "fid:%s furl:%q", f.ID(), f.URL())
 
 	v := `() => document.title`
-	return gojaValueToString(f.ctx, f.Evaluate(v))
+
+	// TODO: return error
```

Comment:
Does this need to be tackled in this PR or another? Maybe we need a new issue for this?